httpclient5/src/main/java/org/apache/hc/client5/http/cookie/StandardCookieSpec.java

```diff
@@ -28,33 +28,31 @@
 package org.apache.hc.client5.http.cookie;
 
 /**
- * Supported cookie specifications.
+ * Cookie specifications by their names supported by the HttpClient.
  *
  * @since 4.3
  */
-public enum CookieSpecs {
+public final class StandardCookieSpec {
```

Comment:
@michael-o Just out of curiosity, why singular case if it is not an enum? Seems to go counter to Oracle's naming scheme in JRE classes (`StandardCharsets` for instance).

Reply:
Valid question. We have throughout the codebase now singular name classes/enums for some kind of identity. If that would be an enum, it would be similar. I have chosen it to be a final class because we need string at the end. I have picked up the idea of `StandardCharsets`, but not ultimately the naming scheme. We now have a consistent naming scheme, imho, throughout the entire code base. Enums for exhaustive identities, final classes for non-exhaustive.

I can update the commit message to make that clear if you want.

Reply:
@michael-o So, what you now see as more consistent I personally see as much less consistent. But so be it.

Reply:
Do you want me to leave the message as-is or add more text?

Reply:
@michael-o Makes no difference as far as I am concerned.